psycopg/psycopgmodule.c

```diff
@@ -899,6 +914,7 @@ INIT_MODULE(_psycopg)(void)
     /* set some module's parameters */
     PyModule_AddStringConstant(module, "__version__", PSYCOPG_VERSION);
     PyModule_AddStringConstant(module, "__doc__", "psycopg PostgreSQL driver");
+    PyModule_AddIntConstant(module, "__libpq_version__", PG_VERSION_NUM);
```

Comment:
It could be a good idea actually. I actually never noticed that PQserverVersion is decimal while ours is hex :)